app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DisplayDatasetActivityTest.kt

```diff
@@ -136,29 +134,9 @@ class DisplayDatasetActivityTest {
         }
     }
 
-    @Test
-    fun modifyingDatasetNameWorks() {
-        launchFragment()
-        val newName = "new dataset name"
-        runBlocking {
-            waitFor(1) // increase if needed
-            onView(withId(R.id.display_dataset_name)).perform(clearText(), typeText("$newName\n"))
-            onView(withId(R.id.display_dataset_name)).check(matches(withText(containsString(newName))))
-
-            // need to get again because dataset is immutable and editing the name creates a new
-            // Dataset object in the database

```

Comment:
Subjective, but I would find it more intuitive if modifying the dataset name wasn't in the modify categories menu item. We could either add a separate menu item, or do it in datasetoverview maybe?